arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,764 @@
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The database account delete operation will complete asynchronously."
+          },
+          "204": {
+            "description": "The specified account does not exist in the subscription."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/failoverPriorityChange": {
+      "post": {
+        "operationId": "DatabaseAccounts_FailoverPriorityChange",
+        "description": "Changes the failover priority for the Azure DocumentDB database account. A failover priority of 0 indicates a write region. The maximum value for a failover priority = (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "failoverParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FailoverPolicies"
+            },
+            "description": "The new failover policies for the database account."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. The failover policy change operation will complete asynchronously."
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/databaseAccounts": {
+      "get": {
+        "operationId": "DatabaseAccounts_List",
+        "description": "Lists all the Azure DocumentDB database accounts available under the subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountsListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
>"nextLinkName": null [](start = 10, length = 20)

Are you sure there will be no more than one page of result for any subscription? if server side exposed ability to get several pages of results then you put developers in the situation that they will not be able to get second page easily.